app/config/config.php

```diff
@@ -69,6 +69,7 @@
     ],
     'session'         => [ // handler_id set to null will use default session handler from php.ini
         'handler_id'           => null,
+        'storage_factory_id'   => 'session.storage.factory.native',
```

Comment:
@biozshock why is this change needed here, if it's only impacting tests?

Reply:
@mollux Because otherwise tests will test wrong configuration. See: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1144